exasol/analytics/schema/dboperation_type.py

```diff
@@ -4,7 +4,8 @@
 )
 
 
-class DbOperationType(Enum):
+# class DbOperationType(Enum):
+class DbSpanType(Enum):
```

Comment:
This is something you probably have discussed, but from the bystander's point of view it's not at all obvious why the operation type is now called span type.

Reply:
Understood - @tkilias already proposed to 
1. revert renaming back to `DBOperationType` 
2. while keeping the name `SPAN_TYPE` in the Audit table 
3. insert the value of `DBOperationType` into column `SPAN_TYPE` in the Audit Table 

> We should keep this the DBOperationType, but we don't store the value in db_operation_type column , but in the db_span_type column